asu/build.py

```diff
@@ -300,6 +301,21 @@ def download_file(filename: str, dest: str = None):
             cache_workdir / ".config",
         )
 
+    # Prevent DoS by large int<->str conversions (https://github.com/python/cpython/issues/95778)
+    # This sets a hard limit of < 1TB as the max processible rootfssize (with the actual limit set in config)
+    if req.get("rootfssize") and len(req.get("rootfssize")) < 7:
```

Comment:
I think this requires more logic now, the rootfssize should be reset after (or before) each build.